config/sara/performance_jurisdictions.yml

```diff
@@ -0,0 +1,962 @@
+            threshold_operator: 'Equal'
+            group: 1
+        'New Loss of Taste':
+            value: true
+            type: 'BoolSymptom'
+            required: true
+            threshold_operator: 'Equal'
+            group: 1
+        'Shortness of Breath':
+            value: true
+            type: 'BoolSymptom'
+            required: true
+            threshold_operator: 'Equal'
+            group: 1
+        'Fever':
+            value: true
+            type: 'BoolSymptom'
+            notes: 'Feeling feverish or have a measured temperature at or above 100.4°F/38°C'
+            required: true
+            threshold_operator: 'Equal'
+            group: 2
+        'Used A Fever Reducer':
+            value: true
+            type: 'BoolSymptom'
+            notes: 'In the past 24 hours, have you used any medicine that reduces fevers?'
+            required: false
+            threshold_operator: 'Equal'
+            group: 2
+        'Chills':
+            value: true
+            type: 'BoolSymptom'
+            required: true
+            threshold_operator: 'Equal'
+            group: 2
+        'Repeated Shaking with Chills':
+            value: true
+            type: 'BoolSymptom'
+            required: true
+            threshold_operator: 'Equal'
+            group: 2
+        'Muscle Pain':
+            value: true
+            type: 'BoolSymptom'
+            required: true
+            threshold_operator: 'Equal'
+            group: 2
+        'Headache':
+            value: true
+            type: 'BoolSymptom'
+            required: true
+            threshold_operator: 'Equal'
+            group: 2
+        'Sore Throat':
+            value: true
+            type: 'BoolSymptom'
+            required: true
+            threshold_operator: 'Equal'
+            group: 2
+        'Nausea or Vomiting':
+            value: true
+            type: 'BoolSymptom'
+            required: true
+            threshold_operator: 'Equal'
+            group: 2
+        'Diarrhea':
+            value: true
+            type: 'BoolSymptom'
+            required: true
+            threshold_operator: 'Equal'
+            group: 2
+        'Fatigue':
+            value: true
+            type: 'BoolSymptom'
+            required: true
+            threshold_operator: 'Equal'
+            group: 2
+        'Congestion or Runny Nose':
+            value: true
+            type: 'BoolSymptom'
+            required: true
+            threshold_operator: 'Equal'
+            group: 2
+        'Pulse Ox':
+            value: 90
+            threshold_operator: 'Less Than'
+            type: 'FloatSymptom'
+            required: true
+    children:
+        'Alabama':
+            phone: '+13455555555'
+            webpage: 'www.example.com'
+            email: 'contact@example.com'
+            message: 'This Is a Custom Jurisdiction Message'
+            send_digest: true
+            children:
+                'Alabama County 1':
```

Comment:
Thoughts on adding sub jurisdictions to counties? If I'm doing the math correctly, I think we have about 150 jurisdictions in this yml which is definitely better than the dev one, but wasn't sure if we were trying to get jurisdiction numbers closer to production where it's probably ~600 by now? I think the depth of jurisdictions in prod is also at least 4, with some sub jurisdictions of states having multiple groups and whatnot. This isn't something that important though and if we're not trying to scale to that level for the purposes here, that's perfectly fine

Reply:
Yes good point on the depth. An oversight by me for sure.